modules/ROOT/pages/upgrade-self-managed.adoc

```diff
@@ -0,0 +1,90 @@
+= Upgrade Runtime Fabric for Self-Managed Kubernetes
+
+To upgrade Anypoint Runtime Fabric for Self-Managed Kubernetes, upgrade Runtime Fabric components and the `rtfctl` command-line utility.
+
+== About Component Upgrades
+
+Upgrading Runtime Fabric components means upgrading services that:
+- Facilitate communication to the control plane
+- Provide load balancing
+- Log and forward metrics
+
+When component upgrades are available, you apply the upgrades within Runtime Manager via a button that's visible when an upgrade is available.
+
+Runtime Fabric component upgrades do not require adding a node.
+
+=== Determine the Component Versions You Are Using
+
+From a local workstation that has a Kubeconfig file and can access Kubernetes, or from a support bastion, run the following command:
+```
+.rtfctl version
+```
+
+The output looks something like this:
+
+* The `rtfctl` command-line utility version in the `rtfctl` field
+* The Runtime Fabric component version in the `agent` field
+
+For example:
+```
+$ ./rtfctl version
+COMPONENT    VERSION
+rtfctl       0.3.122
+agent        1.9.6
+kubernetes   1.19.6-eks-49a6c0
+
+```
+
+[NOTE]
+The Runtime Fabric component version that is in use are also displayed in the *Runtime Fabrics* tab in Runtime Manager. If an upgrade is available, Runtime Manager displays an *Upgrade to vx.x* message.
```

Comment:
```suggestion
The Runtime Fabric component version that is in use is also displayed in the *Runtime Fabrics* tab in Runtime Manager. If an upgrade is available, Runtime Manager displays an *Upgrade to vx.x* message.
```

Reply:
It's either the Runtime Fabric component versions that are in use are also displayed.... or The Runtime Fabric component version that is in use is...

I don't know if multiple versions are displayed